app/controllers/cclow/geography/legislations_controller.rb

```diff
@@ -23,7 +23,10 @@ def show
         @sectors = @legislation.laws_sectors.order(:name)
         @keywords = @legislation.keywords.order(:name)
         @responses = @legislation.responses.order(:name)
-        @legislation_events = @legislation.events_with_eventable_title
+        @legislation_events = @legislation.events_with_eventable_title.order(:date)
```

Comment:
@legislation_events doesn't need to be a instance variable, if it's not used in the views.